plugins/sdm/lib/helper/grant_request_helper.py

```diff
@@ -0,0 +1,112 @@
+import json
+import time
+from collections import namedtuple
+import os
+
+from grant_request_type import GrantRequestType
+
+class GrantRequestHelper:
+    __grant_requests = {}
+    folder_path = "./data/grant_requests"
```

Comment:
Please leave a comment here mentioning that we would like to make this configurable in the future.